core/org.testeditor.core/src/main/java/org/testeditor/core/services/impl/TestProjectServiceImpl.java

```diff
@@ -199,10 +222,19 @@ public void unBind(TestServerService testServerService) {
 	private void loadProjectListFromFileSystem() throws SystemException {
 		testProjects = new ArrayList<TestProject>();
 		File[] files = getWorkspaceDirectories();
-		LOGGER.info("Scanning for Projects in: " + Platform.getLocation().toOSString());
+		logger.info("Scanning for Projects in: " + Platform.getLocation().toOSString());
 		for (File file : files) {
 			if (isTestProjectDirectory(file)) {
 				TestProject testProject = createProjectFrom(file);
+				try {
+					if (teamShareService.hasGlobalLock(testProject)) {
```

Comment:
To check if a project is  teamshared is a good pattern. New story for this issue is a good idea.